net-voip/telepathy-haze/telepathy-haze-0.8.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} pypy3 )
```

Comment:
Did you test this with pypy3?

Reply:
Yes. And python is only needed for the build and tests